command_line/show_mask_info.py

```diff
@@ -0,0 +1,24 @@
+# LIBTBX_SET_DISPATCHER_NAME dev.dxtbx.show_mask_info
+
+import sys
+
+from dxtbx.model.experiment_list import ExperimentListFactory
+
+
+def show_mask_info():
+    try:
+        el = ExperimentListFactory.from_filenames(sys.argv[1:])
+    except FileNotFoundError as e:
+        sys.exit(str(e))
+
+    for i in el.imagesets():
+        d = i.get_detector()
+        m = i.get_mask(0)
+        print(f"---- ----")
+        print(d)
+        for j, _m in enumerate(m):
+            print(f"Module {j} has {_m.count(False)} masked pixels of {_m.size()}")
+
+
+if __name__ == "__main__":
+    show_mask_info()
```

Comment:
Get the command line args at this point:
```suggestion
    show_mask_info(sys.argv[1:])
```

Reply:
👆 et seq. - if this is to be useful outside of this context I will move the function to util or something and then just wrap that in a very thin command-line wrapper (which makes sense to YT)

Reply:
Resolved in 01b904ad3fa9ecf840c2fb5f5b35d8a59e9a038a